src/com/carolinarollergirls/scoreboard/json/ScoreBoardJSONListener.java

```diff
@@ -272,7 +272,18 @@ private void initialize(ScoreBoard sb) {
 		updateMap.put("PenaltyCode.Penalty(G)", "Misconduct-Insubordination");
 		updateMap.put("PenaltyCode.Penalty(?)", "Unknown");
 		updateMap.put("PenaltyCode.FO_EXP(FO)", "Foul Out");
-		updateMap.put("PenaltyCode.FO_EXP(EXP)", "Expelled");
+		updateMap.put("PenaltyCode.FO_EXP(EXP-B)", "Expulsion-Back Block");
+		updateMap.put("PenaltyCode.FO_EXP(EXP-A)", "Expulsion-High Block");
+		updateMap.put("PenaltyCode.FO_EXP(EXP-L)", "Expulsion-Low Block");
+		updateMap.put("PenaltyCode.FO_EXP(EXP-E)", "Expulsion-Leg Block");
+		updateMap.put("PenaltyCode.FO_EXP(EXP-F)", "Expulsion-Forearm");
+		updateMap.put("PenaltyCode.FO_EXP(EXP-H)", "Expulsion-Head Block");
+		updateMap.put("PenaltyCode.FO_EXP(EXP-M)", "Expulsion-Multiplayer");
+		updateMap.put("PenaltyCode.FO_EXP(EXP-C)", "Expulsion-Illegal Contact");
+		updateMap.put("PenaltyCode.FO_EXP(EXP-D)", "Expulsion-Direction");
+		updateMap.put("PenaltyCode.FO_EXP(EXP-P)", "Expulsion-Illegal Position");
+		updateMap.put("PenaltyCode.FO_EXP(EXP-N)", "Expulsion-Interference");
+		updateMap.put("PenaltyCode.FO_EXP(EXP-G)", "Expulsion-Misconduct");
```

Comment:
Should we add unknown too? It seems unlikely to come up, but it's possible.

Reply:
I would add unknown for the sake of it, happy to add that line if you'd like?

Reply:
Yeah, it's worth it for consistency I think.

Reply:
Can you add it to your pull so when I merge or Jared does we can capture it?

Reply:
I can create a quick PR after you merge for the sake of simplicity. My PR is big enough as-is.

Reply:
Actually, better idea. https://github.com/rollerderby/scoreboard/pull/62 contains both @official-sounding's commit and this change so you can merge or rebase that one, and close this one.